filters/consistenthash/balancefactor.go

```diff
@@ -0,0 +1,40 @@
+package consistenthash
+
+import (
+	"fmt"
+
+	"github.com/zalando/skipper/filters"
+	"github.com/zalando/skipper/loadbalancer"
+)
+
+type consistentHashBalanceFactor struct {
+	balanceFactor float64
+}
+
+// NewConsistentHashBalanceFactor creates a filter Spec, whose instances
+// set the balancer factor used by the `consistentHash` algorithm to avoid
+// popular hashes overloading a single endpoint
+func NewConsistentHashBalanceFactor() filters.Spec { return &consistentHashBalanceFactor{} }
```

Comment:
We should not forget to add user docs similar to https://github.com/zalando/skipper/blob/master/docs/reference/filters.md#consistenthashkey